src/plugins/codeStyle/diagnosticMessages.ts

```diff
@@ -77,6 +78,12 @@ const messages = {
         code: CodeStyleError.TypeAnnotation,
         message: `${ST} type annotation required`,
         range
+    }),
+    noPrint: (range: Range, severity: DiagnosticSeverity) => ({
+        severity: severity,
+        code: CodeStyleError.NoPrint,
+        message: `${CS} don't use direct Print statements`,
```

Comment:
While we're doing another round of review, maybe:
"Avoid using direct Print statements"?

Reply:
sounds proper